### DISCUSSION TITLE
Is the rich-markdown-editor going to be deprecated?

### DISCUSSION THREAD

**Discussion by dec0dOS** [outline/outline#3253] [General] [answered]
Hello! Thanks for this awesome work!

I've seen that the [rich-markdown-editor](https://github.com/outline/rich-markdown-editor) has been archived. Is it going to be replaced?

**Comment by tommoor** [comment, marked as answer]
Please see the post here:
https://github.com/outline/rich-markdown-editor/discussions/481#discussion-3423060
